af/fastpath/fastpath/reprocessor.py

```diff
@@ -12,12 +12,14 @@
   jsonl files in new S3 bucket e.g.:
     jsonl/{testname}/{cc}/{ts}/00/{jsonlf.name}
   rows in the jsonl database table
-    upsert or insert-but-not-overwrite
+  rows in the fastpath database table
 
 Usage:
 export AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
 PYTHONPATH=. ./fastpath/reprocessor.py ooni-data ooni-data-eu-fra-test --day 2015-1-1
 
+Note: the bundling of measurements into jsonl gz files has to remain deterministic
```

Comment:
It might help to clarify more precisely what is meant by "deterministic" and why we need it.

Do we mean that the same measurements have to end up inside of the same filenames? That they end up in the same s3 paths? Or that the hash of the jsonl.gz files has to be exactly the same across runs?

Reply:
It's what we discussed in the meetings and on Slack.